pkg/controller/operators/olm/apiservices.go

```diff
@@ -164,60 +162,27 @@ func (a *Operator) checkAPIServiceResources(csv *v1alpha1.ClusterServiceVersion,
 		if serviceAccountName == "" {
 			serviceAccountName = "default"
 		}
-		serviceAccount, err := a.lister.CoreV1().ServiceAccountLister().ServiceAccounts(deployment.GetNamespace()).Get(serviceAccountName)
+		_, err = a.opClient.KubernetesInterface().CoreV1().ServiceAccounts(deployment.GetNamespace()).Get(context.TODO(), serviceAccountName, metav1.GetOptions{})
```

Comment:
How horrible is it to pass a context down?

Reply:
Horrible, Todd and I looked at this last week and it would be threaded up to the top, missing literally everywhere in this codebase. Happy to tackle the whole cahuna in a follow-up.